docs/getting-started-on-snowplow-open-source/quick-start-gcp/query-your-data/index.md

```diff
@@ -10,7 +10,7 @@ import TabItem from '@theme/TabItem';
 
 If you were using the `default` or `secure` example scripts unedited in the last section, you will have created a Postgres database where all of your data is stored. Your database will contain the following standard Snowplow schemas:
 
-<Tabs groupId="destination">
+<Tabs groupId="warehouse">
```

Comment:
I think from a consistency point of view it would be better if we followed the same format everywhere. Otherwise this distinction is one more thing to learn for other people writing the docs, and they might make the wrong choice and wonder why it’s not working (or not notice that it’s not working).

So my suggestion would be to add `queryString` everywhere, including the VS Code templates we have somewhere in this repo.

Reply:
Should have gone with my gut, that was what I did originally before scaling it back. :laughing: 

Done, thanks!